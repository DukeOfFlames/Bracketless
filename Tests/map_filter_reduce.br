START
pylib builtins
pylib functools
Â°list = {-3, -2, -1, 1, 2, 3}
#drucke{{builtins.map}{fn {n} { -> n^2 }, list}}
#drucke{{builtins.filter}{fn {n} { -> n > 0 }, list}}
#drucke{{functools.reduce}{fn {n, m} { -> n * m }, list}}
END