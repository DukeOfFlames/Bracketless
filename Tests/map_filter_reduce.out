{9, 4, 1, 1, 4, 9}
{1, 2, 3}
-36
